src/libexpr/eval.cc

```diff
@@ -2661,6 +2673,7 @@ void EvalState::printStatistics()
     topObj["gc"] = {
         {"heapSize", heapSize},
         {"totalBytes", totalBytes},
+        {"cycles", gcCycles},
```

Comment:
```suggestion
        {"gcCycles", gcCycles},
```
"cycles" is too unspecific.